packages/lib/src/components/Card/components/ClickToPay/components/CtPCards/CtPCardsList/CtPCardsList.tsx

```diff
@@ -45,7 +47,7 @@ const CtPCardsList = ({ cards, onChangeCard }: CtPCardsListProps) => {
     }, [data, onChangeCard]);
 
     return (
-        <Field label={i18n.get('ctp.cards.cardSelector')} name="clickToPayCards">
+        <Field name="clickToPayCards" errorMessage={errorMessage}>
```

Comment:
Doesn't need a label anymore?

Reply:
No! Now we are displaying the Selector without the input label